### ISSUE TITLE
Migration needed for all database tables

### ISSUE THREAD

**Issue by dznicol** [shoprgem/shopr#35]
Hi,

I think I am stating the obvious but this project won't work for existing installations until a migration is built to rename all database tables to have a shopr_ prefix.

Migrating will also need go be careful about what version of shoppe is being migrated from so it covers all tables when doing the renames.

Also I think schema.rb and the single bootstrap initial migration are out of sync. Should these files be brought into sync?

**Comment by deanpcmad** [comment]
Yep this will be part of the "Migration from Shoppe" page which I will be working on at some point

**Comment by craft37** [comment]
@dznicol @deanpcmad I wish you good like gyus.
Cause in case I want to develop ecommerce site in rails btw :)

**Comment by deanpcmad** [comment]
I think there is a migration to rename all the tables already

**Comment by deanpcmad** [comment]
I think there is a migration to rename all the tables already

⁣

Sent from BlueMail

​

**Comment by deanpcmad** [comment]
Hmm it looks like Github took a while to process those email responses... I'll delete them
